.github/workflows/CI.yml

```diff
@@ -31,33 +31,33 @@ jobs:
           fetch-depth: 2
       - name: Check frontend files
         id: check-frontend
-        uses: tj-actions/changed-files@v36.4.0
+        uses: tj-actions/changed-files@v37.0.5
```

Comment:
Hi @VIKTORVAV99, 🙌 author of [tj-actions/changed-files](https://github.com/tj-actions/changed-files) here. I just came across your workflow and couldn't help but notice a neat little tweak you could make. We've recently introduced a new `files_yaml` input which supports multiple filters in a single step, while still achieving the same results.

I think you might find it pretty handy! For more details, feel free to have a look at our [documentation](https://github.com/tj-actions/changed-files#using-local-git-history). 

Should you have any questions or need further assistance, please don't hesitate to ask. Have a fantastic rest of your day!

Reply:
Oh nice!
I'll definitely take a look at that when I have the time!

Thanks for the tip and this great action!